src/core/client/stream/tabs/Configure/LiveUpdatesConfig/LiveUpdatesConfigContainer.tsx

```diff
@@ -1,161 +0,0 @@
-
-  return (
-    <Localized id="configure-liveUpdates-disabledSuccess">
-      Live updates are now disabled
-    </Localized>
-  );
-};
-
-const renderMessages = (
-  showSuccess: boolean,
-  onCloseSuccess: () => void,
-  showError: boolean,
-  onCloseError: () => void,
-  error?: string | null,
-  enabled?: boolean
-) => {
-  return (
-    <div
-      className={
-        showError || showSuccess
-          ? styles.messagesVisible
-          : styles.messagesHidden
-      }
-    >
-      {showSuccess && (
-        <CallOut
-          color="success"
-          icon={<Icon size="sm">check_circle</Icon>}
-          title={renderSuccess(enabled)}
-          onClose={onCloseSuccess}
-          visible={showSuccess}
-          aria-live="polite"
-        />
-      )}
-      {showError && (
-        <CallOut
-          color="error"
-          icon={<Icon size="sm">error</Icon>}
-          title={error}
-          onClose={onCloseError}
-          visible={showError}
-          role="alert"
-        />
-      )}
-    </div>
-  );
-};
-
-const LiveUpdatesConfigContainer: FunctionComponent<Props> = ({ story }) => {
-  const {
-    id,
-    settings: {
-      live: { configurable, enabled },
-    },
-  } = story;
-
-  const [showSuccess, setShowSuccess] = useState(false);
-  const [showError, setShowError] = useState(false);
-  const [error, setError] = useState(null);
-
-  const setLiveUpdate = useMutation(SetLiveUpdateMutation);
-  const onClick = useCallback(async () => {
-    try {
-      setShowSuccess(true);
-      await setLiveUpdate({
-        id,
-        settings: {
-          live: { enabled: !enabled },
-        },
-      });
-    } catch (err) {
-      setShowError(true);
-      setError(err.message);
-    }
-  }, [enabled, id, setShowSuccess, setShowError, setError]);
-
-  const onCloseSuccess = useCallback(() => {
-    setShowSuccess(false);
-  }, [setShowSuccess]);
-  const onCloseError = useCallback(() => {
-    setShowError(false);
-  }, [setShowError]);
-
-  if (!configurable) {
-    return null;
-  }
-
-  if (enabled) {
-    return (
-      <section aria-labelledby="configure-disableLiveUpdates-title">
-        <DisableLiveUpdates onClick={onClick} />
-        {renderMessages(
-          showSuccess,
-          onCloseSuccess,
-          showError,
-          onCloseError,
-          error,
-          enabled
-        )}
-      </section>
-    );
-  } else {
-    return (
-      <section aria-labelledby="configure-enableLiveUpdates-title">
-        <EnableLiveUpdates onClick={onClick} />
-        {renderMessages(
-          showSuccess,
-          onCloseSuccess,
-          showError,
-          onCloseError,
-          error,
-          enabled
-        )}
-      </section>
-    );
-  }
-};
-
-const enhanced = withFragmentContainer<Props>({
-  story: graphql`
-    fragment LiveUpdatesConfigContainer_story on Story {
-      id
-      settings {
-        live {
-          configurable
-          enabled
-        }
-      }
-    }
```

Comment:
The fragment `LiveUpdatesConfigContainer_story` is still being used, that's why we got all the build errors.